### PR TITLE
Add Travis setup

### PR DESCRIPTION
**Pull request by yordis**
This will help to automated some tasks such as testing, linters, and docs report.

@boydm you will need to activate Travis for this project it can start working first.

TODO

- [ ] Fix the links if it gets move into the Org
- [ ] Finalize in the versions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: elixir
+sudo: required
+
+elixir:
+  - 1.7.3
```
Comment (harlantwood) [inline .travis.yml]: I believe if you say `1.7` it will pull the latest patch version, generally a good thing IMO.

Are there other Elixir versions you wish to support @boydm?  We should test under all supported versions.  Can be another PR; not to hold this one up.

Comment (yordis) [reply .travis.yml]: @harlantwood @boydm yeah the versions of Elixir and Erlang are up to you @boydm so let me know and I will update the setup

File: mix.exs [hunk 2]
```diff
@@ -24,47 +24,39 @@ defmodule Scenic.Mixfile do
...
-  #
-  # To depend on another app inside the umbrella:
-  #
-  #   {:myapp, in_umbrella: true}
-  #
-  # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:scenic_math, "~> #{@version}"},
+      {:scenic_math, "~> 0.7.0"},
```
Comment (yordis) [inline mix.exs]: @boydm the reason I did this is because some tools can analyze this statically

Comment (boydm) [reply mix.exs]: Looking at all this now.

I'd rather it be {:scenic_math, "~> 0.7"}, so it automatically gets the most recent minor version.

**Comment by yordis** [comment]
It is failing and something tells me is Travis issue since ETS is about memory .... 🤔

**Comment by yordis** [comment]
@boydm why that test would fail? Is that reading from some local file or something?

**Comment by boydm** [comment]
merged
